src/default_panels/GraphCreatePanel.js

```diff
@@ -10,81 +10,122 @@ import {
   TraceSelector,
 } from '../components';
 import {localize} from '../lib';
-import {scopeDefaults, projNames} from 'plotly.js/src/plots/geo/constants';
 
-const geoScopeOptions = Object.keys(scopeDefaults).map(k => ({
-  label: k,
-  value: k,
-}));
-const geoProjectionOptions = Object.keys(projNames).map(k => ({
-  label: k,
-  value: k,
-}));
-
-const GraphCreatePanel = ({localize: _}) => (
-  <TraceAccordion canAdd>
-    <TraceSelector label={_('Plot Type')} attr="type" show />
-
-    <DataSelector label={_('Labels')} attr="labels" />
-
-    <DataSelector label={_('Values')} attr="values" />
-
-    <DataSelector label={_('Locations')} attr="locations" />
-
-    <DataSelector label={_('Latitude')} attr="lat" />
-
-    <DataSelector label={_('Longitude')} attr="lon" />
-
-    <DataSelector label={_('X')} attr="x" />
-
-    <DataSelector label={_('Y')} attr="y" />
-
-    <DataSelector label={{choropleth: _('Values'), '*': _('Z')}} attr="z" />
-
-    <DataSelector label={_('Color')} attr="marker.color" />
-
-    <DataSelector label={_('Open')} attr="open" />
-
-    <DataSelector label={_('High')} attr="high" />
-
-    <DataSelector label={_('Low')} attr="low" />
-
-    <DataSelector label={_('Close')} attr="close" />
-
-    <Radio
-      label={_('Transpose')}
-      attr="transpose"
-      options={[{label: _('No'), value: false}, {label: _('Yes'), value: true}]}
-    />
-
-    <Section name={_('Location Format')}>
-      <Dropdown
-        attr="locationmode"
-        clearable={false}
+const GraphCreatePanel = ({localize: _}) => {
+  const geoProjectionOptions = {
+    usa: [
+      {
+        label: _('Albers USA'),
+        value: 'albers usa',
+      },
+    ],
+    other: [
+      {label: _('Equirectangular'), value: 'equirectangular'},
```

Comment:
Same here, can we push this all into a `ProjectionsDropdown` and a `GeoScopeDropdown` ?

Reply:
An additional advantage of this approach is that we could get rid of `LayoutSectionOverride` by just binding those two custom fields to the Layout.